src/components/Map/CircleComponent.tsx

```diff
@@ -0,0 +1,31 @@
+import L from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Circle, Marker } from 'react-leaflet';
+
+import './CircleComponent.scss';
+
+function CircleComponent({ center, radius, text, color }) {
```

Comment:
```suggestion
type CircleComponentProps = {
  center: [number, number];
  radius: number;
  text: string;
  color: string;
};

function CircleComponent({ center, radius, text, color }: CircleComponentProps) {
```
You should use `CircleComponentProps` type defined with TypeScript instead of `propTypes`.

I removed the `import './CircleComponent.scss';` in the suggestion because you can apply all styles from this stylesheet file using `tailwind` and remove the `CircleComponent.scss` as it will be no longer needed.

Reply:
You will also have to add this rule
`"@typescript-eslint/member-delimiter-style": "off"`
to the `.eslintrc` file to fix Prettier and eslint conflict with semicolons in the type definition.

This rule should be added in the `rules` field on the configuration JSON object.
```
{
  ...
  "rules": {
    ...
    "@typescript-eslint/member-delimiter-style": "off"
  }
}
```